ext/xmlreader/php_xmlreader.stub.php

```diff
@@ -33,77 +33,77 @@ class XMLReader
     public string $xmlLang;
 
     /** @return bool */
-    public function close() {}
+    public function close() {} // TODO make the return type void
```

Comment:
the same problem as in a previous PR :( (and there is one more of this in ext/xsl)